src/annotator/range-util.js

```diff
@@ -10,6 +10,8 @@ export function isSelectionBackwards(selection) {
   }
 
   const range = selection.getRangeAt(0);
+  // Does not work correctly on iOS when selecting nodes backwards.
+  // https://bugs.webkit.org/show_bug.cgi?id=220523
```

Comment:
Thanks for filing this bug report. I notice that it links to a URL in an S3 bucket that you host. Just in case you remove this bucket in future for any reason I would suggest adding a copy of the HTML file as an attachment to the bug report. Also it may be worth mentioning the URL of this PR as additional context for anyone looking at the issue in future.

Reply:
done!